### PR TITLE
3 signup

### PR DESCRIPTION
**Pull request by acookson91**
This work connects to #3

### CODE REVIEW

File: app/controllers/user.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+class MakersBnb < Sinatra::Base
+
+  get '/user/new' do
+    @user = User.new
+    erb :'user/new'
+  end
+
+  post '/user' do
+    @user = User.create(email: params[:email], password: params[:password])
```
Comment (paulalexrees) [inline app/controllers/user.rb]: needs to be User.new if you're using @user.save later on

File: spec/user_management_spec.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
+feature 'User management' do
```
Comment (kylebuttner) [inline spec/user_management_spec.rb]: this should be in spec/features

File: spec/user_management_spec.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    expect(User.count).to eq(1)
+    expect(User.first.email).to eq('jon.snow@nightswatch.com')
+  end
+
+  scenario 'user can\'t sign up without email' do
+    expect do
+      visit '/user/new'
+      fill_in :password, with: 'ghost'
+      click_button 'Sign up'
+    end.not_to change(User, :count)
```
Comment (kylebuttner) [inline spec/user_management_spec.rb]: ```expect(User.count).not_to change(User, :count)```

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+class User
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :email, String, required: true
+  property :password, String, required: true
+
+  validates_presence_of :email
+  validates_uniqueness_of :email
```
Comment (paulalexrees) [inline app/models/user.rb]: If you have `required: true` in the property, you don't need validates_presence_of below
Also you can remove validates_uniqueness_off if you add `unique: true` to the property

**Comment by paulalexrees** [comment]
:shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit:

**Pull request merged**